tests/helpers/test-errors-on-directive-linking.js

```diff
@@ -0,0 +1,122 @@
+/* jshint es3: false, esnext: true */
+/* global jasmine, beforeEach, describe, it, expect, module, inject */
+
+(function () {
+
+    var helpers = window._helpers = window._helpers || {};
```

Comment:
Instantiates `window._helpers` if it's not already setup. Perhaps this would read better?

```js
var helpers = window._helpers = (window._helpers || {});
```